Flag Bad Candidate Facebook pages in Candidate table

SailingSteve opened the pull request:
Fix to wevote/WeVoteServer/issues/863
Consolidated some duplicate code in import_export_facebook/views_admin

SailingSteve committed:
Flag Bad Candidate Facebook pages in Candidate table
Fix to wevote/WeVoteServer/issues/863
Consolidated some duplicate code in import_export_facebook/views_admin

DaleMcGrew commented on candidate/controllers.py:
```diff
@@ -1406,14 +1410,16 @@ def save_image_to_candidate_table(candidate, image_url, source_link, kind_of_sou
+            candidate.we_vote_hosted_profile_image_url_large = we_vote_hosted_profile_image_url_large
+            candidate.we_vote_hosted_profile_image_url_medium = we_vote_hosted_profile_image_url_medium
+            candidate.we_vote_hosted_profile_image_url_tiny = we_vote_hosted_profile_image_url_tiny
         candidate.save()
```
Do we need to save this if not url_is_broken?

SailingSteve committed:
Flag Bad Candidate Facebook pages in Candidate table
Fix to wevote/WeVoteServer/issues/863
Consolidated some duplicate code in import_export_facebook/views_admin

DaleMcGrew commented:
@SailingSteve, I had a new idea that I didn't think of when we first spoke about this issue. When a We Vote admin tries to paste in a new Facebook page to the Candidate Edit page (on our API server's edit candidate page), could you have it check the new URL and remove the "bad_url_link" value if it is a valid Facebook URL?

DaleMcGrew commented:
👍

SailingSteve commented:
This was one of my test cases, I just tested again, and it seems to work.


On 7/23/18 2:13 PM, Dale John McGrew wrote:
>
> @SailingSteve <https://github.com/SailingSteve>, I had a new idea that 
> I didn't think of when we first spoke about this issue. When a We Vote 
> admin tries to paste in a new Facebook page to the Candidate Edit page 
> (on our API server's edit candidate page), could you have it check the 
> new URL and remove the "bad_url_link" value if it is a valid Facebook URL?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/wevote/WeVoteServer/pull/865#issuecomment-407203083>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGmlhX4pg2llcUQQLJbZzbALTASSHzMoks5uJjxrgaJpZM4Vbqgm>.
>
